lib/src/main/scala/nl.knaw.dans.lib.dataverse/DatasetApi.scala

```diff
@@ -290,12 +292,31 @@ class DatasetApi private[dataverse](datasetId: String, isPersistentDatasetId: Bo
     postFileToTarget[FileList]("add", Option(dataFile), Option(Serialization.write(fileMedataData)))
   }
 
+  def awaitUnlock: Try[List[Lock]] = {
```

Comment:
Should be `Try[DataverseResponse[Nothing]]` .

Returns `Failure` if timeout expires, containing `DataverseException` with in the message the list of locks that are still active.

Reply:
Better `Try[Unit]` as there is no interesting response to return at this point.